spec/components/parameters/reports/message/endDate.ts

```diff
@@ -0,0 +1,16 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const endDate: ParameterObject = {
+  name: 'endDate',
+  in: 'query',
+  required: true,
+  description: 'The end of period to looking for entries',
```

Comment:
Acho que "query entries" fica melhor do que "looking for entries", e mesmo que fosse manter "buscar", seria melhor "search" do que "looking", mesmo "looking" tá conjugado errado, deveria ser "look".

Reply:
Ah! Faltou o artigo também no período, deveria ser "of the period".

E ponto final não faria mal também.